source/release-notes.txt

```diff
@@ -20,6 +20,41 @@ Release Notes for |bi|
 .. expect and we don't break the links on the downloads page.
 .. (DOCS-9670 for context)
 
+.. _bi-2-14-4:
+
+|bi| 2.14.4
+-----------
+
+*Released September 17, 2021*
+
+Improvements
+~~~~~~~~~~~~
+
+- Now built with Go 1.16.0. 
+
+- Adds support for MongoDB 5.0.
+
+- Adds the RHEL8 ARM distribution to the download center.
+
+- Adds the containerized Ubuntu 18.04 distribution to the download 
+  center.
```

Comment:
Containerized ubuntu 18.04 doesn't appear separately on the download center -- it's just the same packages that we use for regular ubuntu 18.04. This platform is just added for testing purposes, to ensure that the normal ubuntu 18.04 package works as expected on a containerized ubuntu 18.04.

We could update this bullet to something like "Adds tests for the containerized Ubuntu 18.04 distribution", or just remove it -- either one is fine by me.

Reply:
Sounds good, I'll remove it since we don't mention other testing